wagtail/admin/views/reports/aging_pages.py

```diff
@@ -46,6 +48,7 @@ class AgingPagesView(PageReportView):
         "last_published_by_user",
         "content_type",
     ]
+    any_permission_required = ["add", "change", "publish"]
```

Comment:
Again, not setting `permission_policy` here as it's already set in `PageReportView`